preview/src/index.html

```diff
@@ -18,6 +18,18 @@
 <iframe id="widget" src="./widget.html"></iframe>
 
 <script type="text/javascript">
+
```

Comment:
```suggestion
```

Reply:
Also, rearrange the functions to be called only after they're declared for better readability, like this:

```
func foo() {}

func bar() {
    foo();
}
```

instead of 

```
func bar() {
    foo();
}

func foo() {}
```

Reply:
Some guide lines recommend doing like that, including clean code. Take a look at the third item [here](https://medium.com/oril/10-tips-to-keep-your-code-more-clean-2fa9aafea1cf#:~:text=Methods%20should%20be%20declared%20in,minor%20methods%20at%20the%20end.). Also I remember a previous PR where I was asked to do the method ordination like that :sweat_smile:.

Reply:
There is a linter for the order Denis recommended. It is even a recommended linter, and for good reason, something might call the function in between the declarations:

```ts
func bar() {
    foo();
}

bar(); // This will fail because `foo` is undefined at this point

func foo() {}
```

But this requirement is a quirk of JS and not really a code style guideline, so we explicitly disabled that linter. The recommendation is to leave the higher lever code higher in the file, so you scroll down to see the implementation detail.